Branch-SDK/Branch-SDK/BNCContentDiscoveryManager.m

```diff
@@ -135,14 +135,17 @@ - (void)indexContentWithTitle:(NSString *)title description:(NSString *)descript
         }
         return;
     }
-    if (!title) {
+    if (!title || ![title isKindOfClass:[NSString class]]) {
```

Comment:
The method def here specifically calls for an `NSString`, type checking that seems unnecessary.